frontend/src/metabase/dashboard/containers/DashboardApp.jsx

```diff
@@ -148,17 +162,12 @@ const DashboardApp = props => {
         );
       }
     }
-  }, [isLoadingComplete, showNotification, dashboard?.name]);
+  }, [dashboard?.name, isLoadingComplete, showNotification]);
 
   const onConfirmToast = useCallback(async () => {
     await requestPermission();
-    setIsShowingToaster(false);
   }, [requestPermission]);
 
-  const onDismissToast = useCallback(() => {
```

Comment:
currently, we have no way to programmatically dismiss the toast created via `addUndo`, we can add this back once we extend the `addUndo` API